integration-cli/docker_cli_inspect_test.go

```diff
@@ -38,6 +38,125 @@ func (s *DockerSuite) TestInspectInt64(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestInspectDefault(c *check.C) {
+
+	/*
+		        Both the container and image are named busybox. docker inspect will fetch the container JSON.
```

Comment:
The indentation of this comment (and Some other comments in this PR) looks off / weird. Are you perhaps mixing tabs and spaces?

Also (afaik) we mostly use `//` for these, not `/* ...*/`

(Not really docs related, but I noticed this when reading over the PR)